examples/abundance_matching.yml

```
@@ -23,12 +24,13 @@ tables:
       growth_function: $growth_function
       cosmology: $cosmology
   galaxies:
-    luminosity: !skypy.galaxy.luminosity.herbel_luminosities
+    magnitude: !skypy.galaxy.luminosity.schechter_lf_magnitude
+      redshift: !numpy.repeat [$redshift, 100]
```

```suggestion
      redshift: $redshift
      size: 100
```
I think `schechter_lf_magnitude ` should have an argument `size=None` to handle a case like this where you want to sample multiple luminosities at a fixed redshift. `skypy.utils.random.schechter` already accepts such a size parameter in the correct manner so should be an easy fix.